plugin.video.attactv/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="plugin.video.attactv" name="attac tv" version="1.1.13" provider-name="Jose Antonio Montes (jamontes)">
+<addon id="plugin.video.attactv" name="attac tv" version="1.1.14" provider-name="Jose Antonio Montes (jamontes)">
     <requires>
         <import addon="xbmc.python" version="2.14.0"/>
         <import addon="plugin.video.youtube" version="4.4.1"/>
```

Comment:
This line has to be removed.

Reply:
Hi Rechi,
Do you mean the Youtube import dependency line, as such add-on is no longer available on Gotham and Helix repos?
```suggestion
        <import addon="plugin.video.youtube" version="4.4.1"/>
```
Please, confirm me the line I must remove from the file and so I'll do it.
Thanks a lot for your time and best regards,
jamontes.

Reply:
Yes exactly, remove this line.
```suggestion
```

Reply:
OK, no problem. I'll do it right now.
Thanks a lot for your help and best regards,
jamontes.